Source/ACE/Entity/DebugObject.cs

```diff
@@ -60,32 +60,11 @@ public DebugObject(ObjectGuid guid,  BaseAceObject baseAceObject)
             this.GameData.Value = baseAceObject.Value;
             this.GameData.ItemCapacity = baseAceObject.ItemsCapacity;
 
-            // Put is in for Ripley - these are the fields I want to write that he was concerned with.
-            if ((this.Type & (ObjectType.Creature | ObjectType.LifeStone | ObjectType.Portal)) == 0)
-            {
-                // because this comes from PCAP data - on create we are not animating.
-                this.PhysicsData.AnimationFrame = 0x65;
-
-                // I think this is wrong - we need the weenieClassId from weenie_class   Leaving it for now
-                // TODO: use view to return the correct value.
-                this.WeenieClassid = baseAceObject.AceObjectId;
-
-                // Container will always be 0 or a value and we should write it.
-                // Not sure if the align packs us out with 0's may be redundant Og II
-                this.WeenieFlags |= WeenieHeaderFlag.Container;
-
-                // Creating from a pcap of the weenie - this will be set by the loot generation factory. Og II
-                this.PhysicsData.PhysicsDescriptionFlag &= ~PhysicsDescriptionFlag.Parent;
-                this.GameData.ValidLocations = (EquipMask)baseAceObject.ValidLocations;
```

Comment:
Yep did a quick and dirty to leave out lifestones, portals and creatures.   I knew they could not be contained or equipped.   Also the database for non equipable items puts a 0 which is the makes it safe to load it anyway.   createobject has a pcap with a value in that field  non zero plus the bit flags tells us to act on it.  I don't think you should delete those lines.